src/views/accounts/AccountChangeSecret/Overview/FailedAccountSummary.vue

```diff
@@ -56,7 +56,7 @@ export default {
             }
           },
           account: {
-            label: this.$t('Username'),
+            label: this.$t('Account'),
             formatter: DetailFormatter,
             formatterArgs: {
               can: this.$hasPerm('accounts.view_account'),
```

Comment:
The code appears to be correct and has no evident abnormalities. However, if you want to add some additional context for reference, feel free! I would also like to mention that there's no known current issue from 2025-03-13. If any issues should arise in the future, please inform us so we could update our records accordingly!